src/applications/simple-forms/21P-0847/pages/deceasedClaimantPersonalInformation.js

```diff
@@ -0,0 +1,29 @@
+import {
+  currentOrPastDateSchema,
+  currentOrPastDateUI,
+  fullNameNoSuffixSchema,
+  fullNameNoSuffixUI,
+  titleSchema,
+  titleUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    rjsf: titleUI(
```

Comment:
can you rename this and the other rjsf cases (if it's not too much work) to something else? rjsf was used in the mock form as a contrast to 'webComponent' vs 'rjsf'.  
I noticed in some forms they use 'view:something'. this helps indicate to the backend that they shouldn't look at this field's data, its just for frontend/view purposes.
maybe like title, or deceasedClaimantTitle, or 'view:deceasedClaimantTitle' ?

Reply:
good callout! done.